VARIABLES.md

```diff
@@ -69,3 +70,31 @@
 | secondary_vpcs | List of VPCs to associate with Service Discovery namespace. | list | `<list>` | no |
 | table_param_filter | A regular expression for selecting necessary table parameters for the SNS listener. If the value isn't set, then no table parameters are selected. | string | `` | no |
 | vpc_id | VPC ID. | string | - | yes |
+
+### apiary_assume_roles
+
+A list of maps.  Each map entry describes a role that is created in this account, and a list of principals (IAM ARNs) in other accounts that are allowed  
```

Comment:
" a role that is created in this account"
Do we create the roles we find here or do we require them to already exist? (i.e. something else has created them before setting this). I think it's the latter, in which case I'd suggest rewording this to "a role that exists in this account" or "a role that has already been created in this account"

Reply:
We actually *do* create the roles that are listed here.